elastic/kibana
# Add translation files to CDN assets

## Summary

Part of https://github.com/elastic/kibana/issues/72880

- Generate translation files for all locales (including all internal plugins) during the CDN asset generation task
- Adapt the `rendering` service to use the translation files from the CDN if configured/enabled

### How to test

Connect to the serverless project that was created for the PR, and confirm the translation file is being loaded from the CDN

<img width="907" alt="Screenshot 2024-04-25 at 15 55 23" src="https://github.com/elastic/kibana/assets/1532934/5a6d9110-2e92-41e5-b066-e792e0015134">
